packages/fullstack/Cargo.toml

```diff
@@ -87,7 +87,34 @@ desktop = ["dep:dioxus-desktop", "server_fn/reqwest", "dioxus_server_macro/reqwe
 mobile = ["dep:dioxus-mobile", "server_fn/reqwest", "dioxus_server_macro/reqwest"]
 default-tls = ["server_fn/default-tls"]
 rustls = ["server_fn/rustls", "dep:rustls", "dep:hyper-rustls"]
-axum = ["dep:axum", "dep:tower-http", "server", "server_fn/axum", "dioxus_server_macro/axum", "default-tls"]
+axum_server_fn = [
+    "dep:axum",
+    "server_fn/ssr",
+    "dep:tokio",
+    "dep:dioxus-ssr",
+    "dep:dioxus-isrg",
+    "dep:hyper",
+    "dep:http",
+    "dep:pin-project",
+    "dep:dioxus-cli-config",
+    "dep:async-trait",
+    "server_fn/axum-no-default",
+    "dioxus_server_macro/axum",
+    "default-tls",
+    "dep:parking_lot",
+    "dioxus-interpreter-js",
+]
+axum = [
+    "dep:axum",
+    "dep:tower-http",
+    "server",
+    "server_fn/axum",
+    "axum_server_fn",
+    "dioxus_server_macro/axum",
+    "default-tls",
+    "axum/ws",
```

Comment:
I don't think we use the ws or macros features any more. We could remove them entirely instead of moving them to the full axum feature